changelog/add_min_and_max_to_list_of_false_positives_for_style_symbol_proc.md

```diff
@@ -0,0 +1 @@
+`min` and `max` results in false positives for `Style/SymbolProc` similarly to `select` and `reject` (see https://github.com/rubocop/rubocop/issues/10864). ([@mollerhoj][])
```

Comment:
Can you tweak it?

```suggestion
* [#10884](https://github.com/rubocop/rubocop/issues/10884): `min` and `max` results in false positives for `Style/SymbolProc` similarly to `select` and `reject`. ([@mollerhoj][])
```

Reply:
Hey, that's just a related issue, not the actual issue - So I figured it was better this way?

Reply:
Yeah, below is also OK.

```suggestion
* [#10864](https://github.com/rubocop/rubocop/pull/10864): `min` and `max` results in false positives for `Style/SymbolProc` similarly to `select` and `reject`. ([@mollerhoj][])
```

Reply:
And can you rename the file name to `changelog/fix_add_min_and_max_to_list_of_false_positives_for_style_symbol_proc.md`?

> * Create one file `changelog/{type}_{some_description}.md`, where `type` is `new` (New feature), `fix` or `change`, and `some_description` is unique to avoid conflicts. Task `changelog:fix` (or `:new` or `:change`) can help you.

See: https://github.com/rubocop/rubocop/blob/master/CONTRIBUTING.md#creating-changelog-entries

Reply:
👍